### PR TITLE
Added OpenAPI `removeOperationIdPrefix` test

### PR DESCRIPTION
**Pull request by CharlesDuboisSAP**
## Context
 

The OpenAPI generator has a breaking change with version `7.9.0`.
We revert the version to `7.8.0` and add a test to make sure that we fix it in the future.

### Feature scope:

 
- [x] Revert OpenAPI generator to version `7.8.0`
- [x] Add OpenAPI `removeOperationIdPrefix` test

## Definition of Done

- [x] Functionality scope stated & covered
- [x] Tests cover the scope above
- [ ] ~Error handling created / updated & covered by the tests above~
- [ ] ~Documentation updated~
- [ ] ~Release notes updated~

### CODE REVIEW

File: datamodel/openapi/openapi-generator/src/test/resources/DataModelGeneratorIntegrationTest/remove-operation-id-prefix/input/sodastore.json [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+          },
+          "404": {
+            "description": "Soda product not found"
+          }
+        }
+      }
+    },
+    "/orders": {
+      "post": {
+        "operationId": "foo2.bar2.get",
```
Comment (CharlesDuboisSAP) [inline datamodel/openapi/openapi-generator/src/test/resources/DataModelGeneratorIntegrationTest/remove-operation-id-prefix/input/sodastore.json:248]: Order also has operation get

File: datamodel/openapi/openapi-generator/src/test/resources/DataModelGeneratorIntegrationTest/remove-operation-id-prefix/output/com/sap/cloud/sdk/services/builder/api/SodasApi.java [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+        /**
+     * <p>Get all soda products</p>
+     * <p></p>
+     * <p><b>200</b> - A list of soda products
+     * @return List&lt;SodaWithId&gt;
+     * @throws OpenApiRequestException if an error occurs while attempting to invoke the API
+     */
+    @Nonnull
+    public List<SodaWithId> get() throws OpenApiRequestException {
```
Comment (CharlesDuboisSAP) [inline datamodel/openapi/openapi-generator/src/test/resources/DataModelGeneratorIntegrationTest/remove-operation-id-prefix/output/com/sap/cloud/sdk/services/builder/api/SodasApi.java:71]: Sodas has operation get

File: pom.xml [hunk 3]
```diff
@@ -109,7 +109,7 @@
 		<maven-plugin-annotations.version>3.15.0</maven-plugin-annotations.version>
 		<maven-plugin-testing.version>3.3.0</maven-plugin-testing.version>
 		<caffeine.version>3.1.8</caffeine.version>
-		<openapi-generator.version>7.9.0</openapi-generator.version>
+		<openapi-generator.version>7.8.0</openapi-generator.version>
```
Comment (newtork) [inline pom.xml:112]: (Comment)

Noteworthy dependency downgrade.

File: datamodel/openapi/openapi-generator/src/test/java/com/sap/cloud/sdk/datamodel/openapi/generator/DataModelGeneratorIntegrationTest.java [hunk 4]
```diff
@@ -82,10 +82,30 @@ private enum TestCase
...
-            "private",
-            null),;
+            Map
+                .of(
+                    "pojoBuilderMethodName",
+                    "builder",
+                    "pojoBuildMethodName",
+                    "build",
+                    "pojoConstructorVisibility",
+                    "private")),
```
Comment (newtork) [inline datamodel/openapi/openapi-generator/src/test/java/com/sap/cloud/sdk/datamodel/openapi/generator/DataModelGeneratorIntegrationTest.java:92]: (Minor)

Ugly new-lines :/

**Review by newtork — APPROVED** [review]

**Pull request merged**